README.md

```diff
@@ -47,6 +49,51 @@ steps:
       nightly-build: true
 ```
 
+### Self-hosted runners and container jobs
+
+It requires an additional step to bring dependencies needed for the action
+itself. These dependencies are preinstalled on GitHub hosted runners, but a
+self-hosted runner and a docker image may miss them.
+
+Configuring `apt-get` to skip recommended and suggested packages reduces the
+cache size from \~200MiB to \~30MiB.
+
+```yaml
+jobs:
+  myjob:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:bookworm
+
+    env:
+      DEBIAN_FRONTEND: noninteractive
+
+    steps:
+      - name: Configure apt-get
+        run: |
+          mkdir -p /etc/apt/apt.conf.d
+          printf '%s\n%s\n'                    \
+            'APT::Install-Recommends "false";' \
+            'APT::Install-Suggests "false";'   \
+            > /etc/apt/apt.conf.d/no-recommends-no-suggests.conf
+
+      - name: Update repositories metadata
+        run: |
+          apt-get -y update
+
+      - name: Workaround interactive tzdata configuration problem (gh-50)
+        run:
+          apt-get -y install tzdata
+
+      - name: Install setup-tarantool dependencies
+        run: |
+          apt-get -y install sudo lsb-release gnupg ca-certificates rsync
+
+      - uses: tarantool/setup-tarantool@ab69f5679e6ea7e5872d9e0901f681587fd29be6
```

Comment:
Can we use a tag here, not the exact commit?

Reply:
I plan to update all the examples to `@v3` before the next release.

Reply:
Ok, let's just not forget this)

Reply:
Sure :)